pkgs/applications/science/logic/monosat/default.nix

```diff
@@ -23,7 +23,7 @@ let
   core = stdenv.mkDerivation {
     name = "${pname}-${version}";
     inherit src;
-    buildInputs = [ cmake zlib gmp jdk8 ];
+    buildInputs = [ cmake zlib gmp jdk8 git ];
```

Comment:
git shouldn't be needed here
* https://github.com/sambayless/monosat/blob/fc2d4adc51b10150473b79646894aee687b6e310/CMakeLists.txt#L22

A flag `-DSHOW_GIT_VERSION=OFF` could be used.

And we fetch from an archive so `.git` isn't available so it doesn't do anything
* https://github.com/sambayless/monosat/blob/fc2d4adc51b10150473b79646894aee687b6e310/CMakeLists.txt#L345

Reply:
You're right. I thought I remember seeing error messages from git not being installed at some point, but I must have been doing something different when that happened.

I've removed the commit that adds git.